contracts/TokenLockerOnHarmony.sol

```diff
@@ -43,6 +43,10 @@ contract TokenLockerOnHarmony is TokenLocker, OwnableUpgradeable {
             lightclient.VerifyReceiptsHash(blockHash, rootHash),
             "wrong receipt hash"
         );
+        require(
+            lightclient.isVerified(uint256(blockHash)),
```

Comment:
This would require the actor (user or some client) on the other side to wait for ~1.5 minute before submitting the proof. Any chance this can be submitted faster while maintaining the security?

Reply:
We could use numberOfBlockConfirmations to check any threshold of block confirmations we would like to. Perhaps 5 confirmations instead of 25 if the speed is a concern? I think we should have some threshold for confirmations greater than 1 to prevent attacks. Thinking about it now as well, checking the difficulty increase since that block was mined as well could be a good idea. Otherwise couldn't someone quickly mine X blocks with a very low difficulty before and add them before the next block comes in?

Reply:
Yes - 25 would be ~6min (not 1.5 minute as I originally stated). But 5 may be too risky to release funds. But on the other hand a 6-min confirmation time isn't too bad compared to various high-security bridges in production out there. Perhaps the best course of action for now is to keep it as it is, and systematically improve later

Reply:
25 is a good number as there is an explanation for it in one of the vitalk's blog post. further, many bridges use 25. let's use it for now.